backend/controllers/iepHistory.js

```diff
@@ -10,6 +33,7 @@ const getIepHistoryFactory = elite2Api => {
     // calls to the database.
     const uniqueUserIds = [...new Set(iepSummary.iepDetails.map(details => details.userId))]
     const uniqueAgencyIds = [...new Set(iepSummary.iepDetails.map(details => details.agencyId))]
```

Comment:
do you need any sorting on establishments or levels? or is the order in which they are presented in the select specified in the web layer?  

(There might be a very slight quirk around having an extra establishment or level in the dropdown which doesn't match any of the details if the summary detail has a level or establishment that isn't shared by any of the other details.. it's very minor, could possibly be fixed by moving the `const [summaryDetail, ...iepDetails]`to just after the data load and using it in each of these maps.)